### ISSUE TITLE
Whole Number - Duration description not accurate and clear

### ISSUE THREAD

**Issue by Dlutchy** [MicrosoftDocs/powerapps-docs#3367]
[Enter feedback here]
I believe that the description for section **Whole Number** - **Duration** is not accurate and clear. A better description could be:

A number value that contains time intervals. A user can select a pre-defined value from the list or manually entered in the format: “x minutes”, “x hours” or “x days”. Hours and days can be entered using decimals, for example, “n.n hours” or “n.n days”. The minutes value must be expressible in whole numbers. Sub-minute values will be rounded to the nearest minute.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 933ab4a6-2cf0-a627-5443-55c590b3ea7c
* Version Independent ID: 2e01f1bc-3f57-deae-4588-4017173fe3df
* Content: [Create and edit columns in Dataverse using Power Apps - Power Apps](https://docs.microsoft.com/en-us/power-apps/maker/data-platform/create-edit-field-portal)
* Content Source: [powerapps-docs/maker/data-platform/create-edit-field-portal.md](https://github.com/MicrosoftDocs/powerapps-docs/blob/main/powerapps-docs/maker/data-platform/create-edit-field-portal.md)
* Service: **powerapps**
* Sub-service: **dataverse-maker**
* GitHub Login: @Mattp123
* Microsoft Alias: **matp**

**Comment by bishalgoswami** [comment]
Thanks for your comment, @Dlutchy. This issue has been assigned to the doc author for review.

**Comment by Mattp123** [comment]
@Dlutchy - I've updated the article and addressed some of your feedback. I made a few other changes to the table and added a screenshot of a whole number column using the duration format. Below is the text that will be published shortly. 

A number value presented as a drop-down list that contains time intervals. A user can select a pre-defined value from the list or type an integer value using the format: “x minutes”, “x hours” or “x days”. Hours and days can be entered using decimals, for example, “1.2 hours” or “1.5 days”. Minute values entered must be expressible in whole minutes. Entering a decimal to represent sub-minute values will be rounded to the nearest minute.

Closing issue.
